src/executables/util_service.c

```diff
@@ -1770,10 +1770,13 @@ process_server (int command_type, int argc, char **argv, bool show_usage, bool c
 		{
 		  int pid;
 
-		  if (server_type_config::SINGLE_NODE ==
+		  if (util_get_ha_mode_for_sa_utils () != HA_MODE_OFF
```

Comment:
Q. Why can't we use HA_DISABLED  or HA_GET_MODE ()?  I'm asking since `util_get_ha_mode_for_sa_utils` seems to mean it's for SA mode. I have to say that I don't really know the relevant code.

Reply:
Analysis : 
cubrid utility works as SA_MODE. (util_service.c)
So, if `ha_mode` is set to `ON`, then it sets `PRM_ID_HA_MODE_FOR_SA_UTILS_ONLY` to `ON` and use this parameter. (`util_get_ha_mode_for_sa_utils ()` can get the value for this parameter)

Reply:
Thank you for the information!